api/controllers/blocks.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+const blocks = require('../models/blocks');
+const users = require('../models/users');
+
+const endpoint = '/blocks';
+
+// create new block
+router.post(endpoint, async (req, res) => {
+  if (!req.body.content) {
+    return res.status(400).json('block content required');
```

Comment:
you can use `send()` instead of `json()` if all you're returning is a string.  save `json()` for objects.